pytensor/graph/basic.py

```diff
@@ -1615,37 +1741,65 @@ def expand(o: Apply) -> List[Apply]:
     )
 
 
-def is_in_ancestors(l_apply: Apply, f_apply: Apply) -> bool:
-    """Determine if `f_apply` is in the graph given by `l_apply`.
+def apply_in_ancestors(
+    apply: Apply, ancestors_of: Union[Apply, Collection[Apply]]
+) -> bool:
+    """Determine if `f_apply` is in the graph given by `ancestors_of`.
```

Comment:
I think the meaning here is swapped from the `variable_in_ancestors` no? Here we are checking if the collection includes an ancestor of the node, whereas in `variable_in_ancestors` we are checking if the variable is an ancestors of one of the items in the collection?

Reply:
what do you suggest?